opencensus/metrics/export/metric_descriptor.py

```diff
@@ -84,13 +83,13 @@ class MetricDescriptorType(object):
     @classmethod
     def to_type_class(cls, metric_descriptor_type):
         type_map = {
-            cls.GAUGE_INT64: int,
-            cls.GAUGE_DOUBLE: float,
-            cls.GAUGE_DISTRIBUTION: ValueDistribution,
-            cls.CUMULATIVE_INT64: int,
-            cls.CUMULATIVE_DOUBLE: float,
-            cls.CUMULATIVE_DISTRIBUTION: ValueDistribution,
-            cls.SUMMARY: ValueSummary
+            cls.GAUGE_INT64: value.ValueLong,
```

Comment:
I have a general question, would this hurt the performance as we need to build the dictionary each time the function gets called (and optimization doesn't work since `cls` is parameterized)?

Reply:
That's a good point. I assumed it was cheap enough to do that I didn't bother trying to optimize it, but this will get called on every stat conversion and there's no reason not to move this up into the class. Changed in 7a43a50.

Reply:
It'd be interesting to profile the library on an example app to see what other easy improvements we could make.

Reply:
Good point. We should have some level of performance test in the CI.